piptools/scripts/compile.py

```diff
@@ -58,7 +58,7 @@ class PipCommand(pip.basecommand.Command):
               help=('Output file name. Required if more than one input file is given. '
                     'Will be derived from input file otherwise.'))
 @click.option('--allow-unsafe', is_flag=True, default=False,
-              help="Pin packages considered unsafe: pip, setuptools & distribute")
+              help="Pin packages considered unsafe: {}".format(list(UNSAFE_PACKAGES)))
```

Comment:
I did consider formatting the output but figured the overhead wasn't worth it since we wouldn't be able to have it identical to the original and the list output is still consumable. Happy to update if you think it's worth it @jdufresne